### PR TITLE
Rtcp parser

### PR DESCRIPTION
**Pull request by bblaszkow06**

### CODE REVIEW

File: lib/membrane/rtp/session/receive_bin.ex [hunk 1]
```diff
@@ -24,17 +24,18 @@ defmodule Membrane.RTP.Session.ReceiveBin do
...
               custom_depayloaders: [
-                spec: %{RTP.payload_type() => module()},
+                spec: %{RTP.payload_type_t() => module()},
                 default: %{},
                 description: "Mapping from a payload type to a custom depayloader module"
               ]
 
-  def_input_pad :input, demand_unit: :buffers, caps: :any, availability: :on_request
+  def_input_pad :rtp_in, demand_unit: :buffers, caps: :any, availability: :on_request
+  def_input_pad :rtcp_in, demand_unit: :buffers, caps: :any, availability: :on_request
```
Comment (mat-hek) [inline lib/membrane/rtp/session/receive_bin.ex:38]: I'd consider `:input` and `:rtcp_input` names

File: lib/membrane/rtcp/parser.ex [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
   use Membrane.Filter
 
+  alias Membrane.Buffer
   alias Membrane.RTCP
 
-  def_input_pad :input, caps: RTCP, demand_unit: :buffers
+  @type notification_t() :: {:received_rtcp, RTCP.CompoundPacket.t()}
 
-  # TODO: implement the element
+  def_input_pad :input, caps: :any, demand_unit: :buffers
```
[the remark this answers is not in the document]
Comment (bblaszkow06) [reply lib/membrane/rtcp/parser.ex:14]: Take a closer look, there's no need for `handle_demand`, there's no output pad.

[the remark this answers is not in the document]
Comment (mat-hek) [reply lib/membrane/rtcp/parser.ex:14]: So why is it a filter?

[the remark this answers is not in the document]
Comment (mat-hek) [reply lib/membrane/rtcp/parser.ex:14]: so why is it a filter?

**Pull request merged**